salt/journal/config/etc-nginx-sites-enabled-journal-local-demo.conf

```diff
@@ -11,15 +11,16 @@ server {
     }
 
     location / {
-        try_files $uri /app_{{ environment_name }}.php$is_args$query_string;
+        try_files $uri /app.php$is_args$query_string;
     }
 
-    location ~ ^/app_{{ environment_name }}\.php(/|$) {
+    location /app.php {
```

Comment:
some of these `location` have lost the final `/` but I guess it's unnecessary now that the match is being done with a prefix